### ISSUE TITLE
Plugin not initializing on `plugins_loaded` action

### ISSUE THREAD

**Issue by corywebb** [mtekk/Breadcrumb-NavXT#207]
The `plugins_loaded` action isn't getting triggered in the plugin, so the `bcn_init` function is never called and the plugin is never initialized properly.

Version 6.1.0
WP Version: 4.9.7

**Comment by mtekk** [comment]
Can you elaborate/provide more information as to the behavior you are seeing? If the `bcn_init` function doesn't run, Breadcrumb NavXT does not really load (no settings page, no widget, and the `bcn_display*` functions will result in PHP warnings/errors being thrown). The only reason, that I can think of, for this to happen is if your PHP version is less than 5.3.0 (which is not supported and has not been for a long time).

**Comment by corywebb** [comment]
Hi @mtekk, thanks for responding so quickly.

It's happening in the WordPress.com VIP Go platform as well as in my local install in [VVV](https://varyingvagrantvagrants.org/), which is running PHP 7.0.

The plugin just isn't working. The settings page isn't available on the back end, and on the front end the breadcrumbs are not displaying.

I put `error_log()` in the code to write logs to `debug.log` and see what's happening. Our call to `bcn_display` isn't getting past `if($breadcrumb_navxt !== null)`. I put an `error_log()` in `bcn_init` and it isn't even being called. I put `error_log()` before and after `do_action( 'plugins_loaded' );` in `wp-settings.php` to verify that the action is actually triggered, and it is.

I hope that helps. It's just that for some reason, the `plugins_loaded` action isn't getting triggered in `breadcrumb-navxt.php` and calling `bcn_init()`. I'll do a little more debugging and see if I can figure out why.

**Comment by corywebb** [comment]
Just to add, it looks like the `plugins_loaded` action is done _before_ the `breadcrumb-navxt.php` file is getting loaded, so the plugin is getting loaded _after_ the hook.

**Comment by mtekk** [comment]
Ok, that's interesting. I had heard WordPress VIP does things in a special manner (I've never worked with it myself). If functions.php files are allowed for you theme (I honestly don't know what VIP allows you to do), you could try calling `wpcom_vip_load_plugin` in a function attached to the 'plugins_loaded` action at a priority of 10, I believe that *should* maintain the normal functionality of Breadcrumb NavXT.

**Comment by corywebb** [comment]
Yes, you are allowed a `functions.php` file in VIP. Unfortunately, it loads _after_ the `plugins_loaded` hook. So even trying to use `plugins_loaded` in the theme wouldn't work.

Their new platform, VIP Go, is a lot more flexible, so there are a couple of ways around this issue. I could enable the plugin via the admin in the normal way, or load it in the mu-plugins folder to force the plugin to load without having to enable it.

So, long story short, this isn't an issue with Breadcrumb-NavXT. :)
